### DISCUSSION TITLE
Build a Flutter app with Very Good CLI and Supabase

### DISCUSSION THREAD

**Discussion by jtkeyva** [supabase/supabase#7527] [Show and tell]
Just came across this. I use VG CLI with Supabase so it's nice to see an official article from the source!

https://verygood.ventures/blog/flutter-app-very-good-cli-supabase

@dshukertjr thought you might find this cool :)

**Comment by dshukertjr** [comment]
@jtkeyva Yeah, this was very exciting! I love their work, and I am a heavy user of very good CLI, and bloc as well, so this was massive! Thanks for sharing it with us!

**Reply by jtkeyva** [reply]
Yeah, it's like a great opinionated yet battle tested starter boilerplate to get people to jump right into flutter & supabase :)

**Reply by dshukertjr** [reply]
@jtkeyva 
Yeah, certainly!

I have added some sample SQL on the readme on Happy Chat Flutter repo to get started. It might be slightly different from the ones used in the stream, but should cover the same functionality without modifying anything on the Flutter app!

https://github.com/dshukertjr/happy_chat_flutter

**Reply by jtkeyva** [reply]
very cool thanks! i think that is a great launching pad that will help people dive right in to realtime
